docs/tutorials/adding-new-project.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Adding a new project
+subject: Projects
+---
+
+We currently have a [projects page] where you can browse projects that have
+been built using ev3dev. You can add your own too!  Each project gets a
+dedicated page for the author to explain what they have been working on, as
+well as provide videos, pictures, build instructions, code, and any other media
+that pertains to the project. This is where we explain how you can submit your
+project page to be hosted on the site.
+
+##Overview
+
+All the projects on the website are stored in the [projects folder].
+Each post is saved as a Markdown file, and is automagically converted to HTML
+when you publish your changes (read more about Markdown
+[here][markdown basics]). To submit a page,
```

Comment:
The website actually uses the Kramdown markdown engine rather that GitHub flavored markdown. Here, I think we could say "Kramdown" instead of "Markdown" and use this link: <http://kramdown.gettalong.org/quickref.html>.